source/features/copy-on-y.tsx

```diff
@@ -11,12 +11,8 @@ const handler = ({key, target}: KeyboardEvent): void => {
 	}
 };
 
-function init(): VoidFunction {
-	window.addEventListener('keyup', handler);
-
-	return () => {
-		window.removeEventListener('keyup', handler);
-	};
+function init(signal: AbortSignal): void {
+	window.addEventListener('keyup', handler, {signal});
```

Comment:
Should this be using `delegate`?

Reply:
`delegate` is `addEventListener` + `closest`. Here we're only using an `isEditable` check to _exclude_ specific elements, which doesn't work the same as as `delegate`